test/api/buffer/exports/update/005-update-mapping-bucket-changed-ok/expected-response.json

```diff
@@ -0,0 +1,25 @@
+{
+  "id": "my-export-1",
+  "receiverId": "my-receiver",
+  "name": "My Export 1 Updated",
+  "mapping": {
+    "tableId": "in.c-bucket2.table1",
```

Comment:
Vsimol som si, ze nemame nikde test na zmenu `bucket` (iba tabulky), kedze si to vyzaduje zalozenie novej bucket, tak som pridal tento request do existujuceho testu.